app/src/test/java/io/crate/plugin/PluginLoaderTest.java

```diff
@@ -104,13 +104,6 @@ public void testLoadPluginRegisteringScalarFunction() throws Exception {
                        DataTypes.LONG)));
     }
 
-    @Test
-    public void testLoadPluginWithAlreadyLoadedClass() throws Exception {
```

Comment:
why is this test not needed anymore?

Reply:
There are already other tests which check for the error when loading duplicate through plugins. This one used a JAR which contained `ClusterId` which was removed and thus it was not a duplicate class anymore. Because other tests test this as well, we removed this test.